src/app/components/pie-chart/pie-chart.component.html

```diff
@@ -0,0 +1,13 @@
+<div style="display: block">
+    <canvas baseChart
+            [data]="co2PieChartData"
+            [labels]="pieChartLabels"
+            [chartType]="pieChartType"
+            [colors]="pieChartColors"
+            [legend]="pieChartLegend"
+            (chartHover)="chartHovered($event)"
```

Comment:
Do you have to add this function? There is no hover on mobile devices I think

Reply:
oh right, that came with copy-pasting from the example code ifound online. ;-) I will remove it.